layout/footer/constants.js

```diff
@@ -15,28 +15,6 @@ export const FOOTER_LINKS = [
       }
     ]
   },
-  {
-    id: 'topics',
-    label: 'Topics',
-    route: 'topics',
-    pathnames: ['/app/topics', '/app/topics-detail'],
-    children: [
-      { label: 'Cities', route: 'topics_detail', params: { id: 'cities' } },
-      { label: 'Climate', route: 'topics_detail', params: { id: 'climate' } },
-
-      { label: 'Energy', route: 'topics_detail', params: { id: 'energy' } },
-      { label: 'Food', route: 'topics_detail', params: { id: 'food' } },
-      { label: 'Forests', route: 'topics_detail', params: { id: 'forests' } },
-      { label: 'Society', route: 'topics_detail', params: { id: 'society' } },
-      { label: 'Oceans', route: 'topics_detail', params: { id: 'oceans' } },
-      { label: 'Water', route: 'topics_detail', params: { id: 'water' } }
```

Comment:
@dancer2090 We should keep the links to the dashboards in the footer as well.

Reply:
Confirmed by daniel.caso:
The footer on the design is a bit outdated. Let’s keep it as it is now, keeping the featured dashboads listed.
We might consider to change it if we start including several more featured dashboards.

Fixed here: https://github.com/resource-watch/resource-watch/pull/1179/commits/1dce21a59d944c0e4c75ab45a95b414ffcf43e65